airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/incidents.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
```

Comment:
Use `number` as type instead of `string`

Reply:
As above, ids from this API are UUIDs:

```
{
    "totalCount": 7,
    "data": [
        {
            "id": "3eb3b201-dd9e-4e12-bc7f-ea7b0ef513a4",
            "description": "",
            "impactedServices": [
                "6d458cb4-805e-4dbf-a990-1f735ec9f1d6"
            ],
            "tinyId": "9",
            "message": "Testing Incident Reporting",
            "status": "open",
            "tags": [],
            "createdAt": "2022-09-16T03:23:34.672Z",
            "updatedAt": "2022-09-16T03:23:35.21Z",
            "priority": "P5",
            "ownerTeam": "",
            "responders": [
                {
                    "type": "user",
                    "id": "a9d98683-7ddf-40da-a68b-2eadb54f16e8"
                }
            ],
            "extraProperties": {},
            "links": {
                "web": "https://zip.app.opsgenie.com/incident/detail/3eb3b201-dd9e-4e12-bc7f-ea7b0ef513a4",
                "api": "https://api.opsgenie.com/v1/incidents/3eb3b201-dd9e-4e12-bc7f-ea7b0ef513a4"
            },
            "impactStartDate": "2022-09-16T03:23:34.672Z",
            "actions": []
        },
       ...
    ],
    "paging": {
        "first": "https://api.opsgenie.com/v1/incidents?limit=20&sort=insertedAt&offset=0&order=desc",
        "last": "https://api.opsgenie.com/v1/incidents?limit=20&sort=insertedAt&offset=0&order=desc"
    },
    "took": 0.053,
    "requestId": "066eb680-4048-4a65-8cf1-bd0c5f9a36c4"
}
```